backend/api/Areas/Project/Models/Dispose/ProjectModel.cs

```diff
@@ -76,7 +76,7 @@ public class ProjectModel : Api.Models.BaseModel
         /// <summary>
         /// get/set - An array of properties associated with this project.
         /// </summary>
-        public IEnumerable<ProjectPropertyModel> Properties { get; set; } = new List<ProjectPropertyModel>();
+        public IEnumerable<ProjectPropertyModel> ProjectProperties { get; set; } = new List<ProjectPropertyModel>();
```

Comment:
Why rename?

Reply:
More accurately reflects the object. Properties are actually child entities of projectProperty. Project Property is really just a join entity between projects(buildings or parcels) and the project.